src/odom_predictor.cpp

```diff
@@ -65,21 +80,23 @@ void OdomPredictor::odometryCallback(const nav_msgs::OdometryConstPtr& msg) {
 void OdomPredictor::imuCallback(const sensor_msgs::ImuConstPtr& msg) {
   if (msg->header.stamp < imu_queue_.back().header.stamp) {
     ROS_ERROR_STREAM("Latest IMU message occured at time: "
-                     << msg->header.stamp
-                     << ". This is before the previously received IMU "
-                        "message that ouccured at: "
-                     << imu_queue_.back().header.stamp
-                     << ". The current imu queue will be reset.");
+                         << msg->header.stamp
+                         << ". This is before the previously received IMU "
+                            "message that ouccured at: "
```

Comment:
s/ouccured/occurred

Reply:
Also fixed the typo (I hope that is what you meant) :shipit: